test/spec/ol/renderer/webgl/pointslayer.test.js

```diff
@@ -592,4 +593,81 @@ describe('ol.renderer.webgl.PointsLayer', function () {
       );
     });
   });
+
+  describe('fires events', () => {
+    let layer, source, renderer, frameState;
+
+    beforeEach(function () {
+      source = new VectorSource({
+        features: new GeoJSON().readFeatures({
+          'type': 'FeatureCollection',
+          'features': [
+            {
+              'type': 'Feature',
+              'properties': {},
+              'geometry': {
+                'type': 'Point',
+                'coordinates': [13, 52],
+              },
+            },
+          ],
+        }),
+      });
+
+      layer = new VectorLayer({
```

Comment:
I would suggest using a webgl-based layer here, the result is exactly the same but it simply is a bit less confusing IMO.

```suggestion
      layer = new WebGLPointsLayer({
        source,
        style: {
          symbol: {
            symbolType: 'square',
          },
        },
      });
```

Reply:
Of course, thank you!
Also thanks for checking the unit test - after reading your comment, I really wasn't sure anymore, either. 
Commit comming up in a second!